Yoast/javascript
# Introducing the Polish stemmer

## Summary

This PR can be summarized in the following changelog entry:

* [yoastseo] adds and refactors a part of the external stemmer for Polish morphology together with a spec file.

## Relevant technical choices:
* A gold standard list based on an existing sample vocabulary was added. In its current form, the stems of this list only represent the output of the stemmer. They have not been checked manually.

## Test instructions
This PR can be tested by following these steps:

* Make sure that all current specs
* Add more words and their expected stems (this corpus can help with finding words with certain endings http://nkjp.pl/poliqarp/nkjp300/ ) 
* Run the coverage test and make sure there are no errors

## Quality assurance


Fixes https://yoast.atlassian.net/jira/software/projects/LIN/boards/4?selectedIssue=LIN-368
